### PR TITLE
Update sidebar for account management

### PR DESCRIPTION
**Pull request by PeloWriter**
This PR reorganizes the information surrounding account management, and includes a pretty heavy edit of the User Settings content (removing content that is largely self-evident from viewing the UI). 

I'm specifically interested in feedback on whether this is technically correct, as it is largely a test of how to restructure the content within our repo. Thank you.

### CODE REVIEW

File: gatsby/src/docs/accounts/user-settings/index.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Account Details
+
+Use account details to set your preferred: 
+
+- **Stack Trace Order**: the default is `let Sentry decide`
+- **Language**
+- **Timezone**
+- Whether to use a **24-hour clock**
+
+You can also upload an Avatar to personalize your acount. Choose among displaying your initials, an image, or a Gravatar next to your name.
```
Comment (MimiDumpling) [inline gatsby/src/docs/accounts/user-settings/index.mdx:17]:
```suggestion
You can also upload an Avatar to personalize your account. Choose among displaying your initials, an image, or a Gravatar next to your name. 
```

File: gatsby/src/docs/accounts/user-settings/index.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ 
+
+Click the **Session History** tab to find a historical list of your sessions, including First Seen and Last Seen timestamps.
+
+## Notifications
+
+Sentry notifications are broadly categorized into alerts and non-alert notifications.
+
+- Alerts notify you about problems in your application and can be sent to many supported [integrations](/workflow/integrations/#webhook-alerts). You can set alerts on both errors and performance data.
+- Non-alert are are all other types of Sentry notifications such as workflow notifications, deploy notification, and quota & usage notifications. These are sent only email.
```
Comment (MimiDumpling) [inline gatsby/src/docs/accounts/user-settings/index.mdx:36]:
```suggestion
- Non-alert are all other types of Sentry notifications such as workflow notifications, deploy notification, and quota & usage notifications. These are sent to only email.
```

File: gatsby/src/docs/accounts/user-settings/index.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## API
+
+### Applications
+
+Integrating your application with Sentry allows you to send events to your Sentry instance automatically. To create an application, click the **Create New Application** button.
+
+### Auth Tokens
+
+Authentication tokens allow you to perform actions against the Sentry API on behalf of your account. To create one, click the **Create New Token** button
```
Comment (MimiDumpling) [inline gatsby/src/docs/accounts/user-settings/index.mdx:84]:
```suggestion
Authentication tokens allow you to perform actions against the Sentry API on behalf of your account. To create one, click the **Create New Token** button.
```

**Comment by dcramer** [comment]
just skimmed, seems good though!

File: gatsby/src/docs/accounts/user-settings/index.mdx (hunk 1 above)
Comment (PeloWriter) [reply gatsby/src/docs/accounts/user-settings/index.mdx:17]: Thank you.

File: gatsby/src/docs/accounts/user-settings/index.mdx (hunk 2 above)
Comment (PeloWriter) [reply gatsby/src/docs/accounts/user-settings/index.mdx:36]: Thank you.

File: gatsby/src/docs/accounts/user-settings/index.mdx (hunk 3 above)
Comment (PeloWriter) [reply gatsby/src/docs/accounts/user-settings/index.mdx:84]: Thank you.

**Pull request merged**